setuptools/tests/test_build_meta.py

```diff
@@ -447,7 +447,11 @@ def test_build_with_pyproject_config(self, tmpdir, setup_script):
             "Requires-Dist: tomli >=1 ; extra == 'all'",
             "Requires-Dist: importlib ; python_version == \"2.6\" and extra == 'all'",
         ):
-            assert line in metadata
+            try:
+                assert line in metadata
+            except AssertionError:
+                print(f"\n\n~~~ Expecting {line!r} to be in\n{metadata}\n\n")
+                raise
```

Comment:
I know this is just a debugging test, but fyi you can get that info even simpler:
```suggestion
            assert line in metadata, f"\n\n~~~ Expecting {line!r} to be in\n{metadata}\n\n"
```

Or even a minimalist
```suggestion
            assert line in metadata, (line, metadata)
```

Reply:
Yeah, to be honest I am disappointed with `pytest` here... it is supposed to show this information already right? But for some reason this is the only hint it is giving:

```
>               assert line in metadata
E               AssertionError
setuptools/tests/test_build_meta.py:451: AssertionError
```